nikola/plugins/command/import_wordpress.py

```diff
@@ -63,6 +49,16 @@
     get_default_translations_dict
 )
 
+try:
+    import html2text
+except ImportError:
+    html2text = None
```

Comment:
This probably needs `# NOQA`, and if it doesn’t, it would also not be needed for `phpserialize` and other instances of try-import-except-None.

Reply:
I just checked this:
* If the import is there to have something available in a module that isn't used (i.e. for backwards compatiblity) you need `# NOQA`. In this case there isn't usually try-except.
* If it is an optional import (handled with try-except) you can remove the `# NOQA`.

I'll be removing it for the latter.